services/drone/drone-build.service.js

```diff
@@ -0,0 +1,123 @@
+'use strict'
+
+const Joi = require('joi')
+const serverSecrets = require('../../lib/server-secrets')
+const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  status: Joi.alternatives()
+    .try(isBuildStatus, Joi.equal('none'))
+    .required(),
+}).required()
+
+module.exports = class DroneBuild extends BaseJsonService {
+  static get category() {
+    return 'build'
+  }
+
+  static get route() {
+    return {
+      base: 'drone',
+      pattern: ':scheme(http|https)?/:host/build/:user/:repo/:branch*',
+    }
+  }
+
+  static get staticPreview() {
+    return {
+      message: 'success',
+      color: 'brightgreen',
+    }
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'build',
+    }
+  }
+
+  static render({ status }) {
+    return renderBuildStatusBadge({ status })
+  }
+
+  async handle({ scheme, host, user, repo, branch }) {
+    const ref = branch ? `refs/heads/${branch}` : undefined
+    const options = { qs: { ref } }
+    if (host === 'cloud') {
+      scheme = 'https'
+      host = 'cloud.drone.io'
+    }
+    if (serverSecrets.drone_token) {
+      options.headers = {
+        Authorization: `Bearer ${serverSecrets.drone_token}`,
+      }
+    }
+    const json = await this._requestJson({
+      schema,
+      url: `${scheme}://${host}/api/repos/${user}/${repo}/builds/latest`,
+      options,
+    })
+    return this.constructor.render({
```

Comment:
Not a big deal, but you could just call the `renderBuildStatusBadge` helper here (and in the examples)

Reply:
Done! 😄